pkg/crc/machine/bundle/metadata.go

```diff
@@ -193,6 +193,8 @@ func (bundle *CrcBundleInfo) GetBundleType() crcPreset.Preset {
 		return crcPreset.Podman
 	case "microshift", "microshift_custom":
 		return crcPreset.Microshift
+	case "okd", "okd_custom":
+		return crcPreset.OKD
```

Comment:
Untested, but I'd rewrite this method as:
```
diff --git a/pkg/crc/machine/bundle/metadata.go b/pkg/crc/machine/bundle/metadata.go
index c06484bb0..a73519e07 100644
--- a/pkg/crc/machine/bundle/metadata.go
+++ b/pkg/crc/machine/bundle/metadata.go
@@ -186,16 +186,11 @@ func (bundle *CrcBundleInfo) GetBundleNameWithoutExtension() string {
 }
 
 func (bundle *CrcBundleInfo) GetBundleType() crcPreset.Preset {
-       switch bundle.Type {
-       case "snc", "snc_custom":
-               return crcPreset.OpenShift
-       case "podman", "podman_custom":
-               return crcPreset.Podman
-       case "microshift", "microshift_custom":
-               return crcPreset.Microshift
-       default:
-               return crcPreset.OpenShift
+       bundleType := strings.TrimSuffix(bundle.Type, "_custom")
+       if bundleType == "snc" {
+               bundleType = "openshift"
        }
+       return crcPreset.ParsePreset(bundleType)
 }
 
 func (bundle *CrcBundleInfo) IsOpenShift() bool {
```

Reply:
I changed for it to look like it, and added tests.
I'm not sure if i did them correctly, this is the first time i do anything for such a big Golang project.

Reply:
@Maurycy-Krzeminski This looks good, can you also squash all those commit to a single one?

Reply:
@praveenkumar perhaps provide instructions around rebase or a link to a doc for this:

In short it is something like
```
$ git rebase -i HEAD~3
```

your editor starts

```
p 609507f0
s bcd5fae1
s 9a36ee18
```

Which means to `p`ick the top one, and `s`quash the other commits in. When you close the editor, it will create a squashed commit that needs a new title and description.

Reply:
@praveenkumar  sure, squashed all into one commit  and forced pushed. 

In txt file there is how i locally started crc with okd preset 
It didn't finish starting but last time i didn't have problems 
[log.txt](https://github.com/crc-org/crc/files/11297774/log.txt)

There was a problem when kubeadmin-password was not set

Reply:
But after coppying kubeconfig from C:\Users\----\.crc\machines\crc to C:\Users\----\.kube\config 
then everything works with oc, and kubectl

PS E:\programming\forks\crc\out\windows-amd64> oc get nodes
NAME                 STATUS   ROLES                         AGE   VERSION
crc-dp64w-master-0   Ready    control-plane,master,worker   51d   v1.25.4+a34b9e9